### ISSUE TITLE
undefined method `development?' for nil:NilClass

### ISSUE THREAD

**Issue by davidmles** [danielbayerlein/middleman-google-analytics#24]
When building my site I'm getting this error:

```
== Request: /page/2/index.html
       error  build/page/2/index.html
undefined method `development?' for nil:NilClass
/Users/david/.rbenv/versions/2.2.4/lib/ruby/gems/2.2.0/gems/middleman-google-analytics-2.0.0/lib/middleman-google-analytics/helpers.rb:46:in `legacy_development?'
/Users/david/.rbenv/versions/2.2.4/lib/ruby/gems/2.2.0/gems/middleman-google-analytics-2.0.0/lib/middleman-google-analytics/helpers.rb:22:in `render_script'
/Users/david/.rbenv/versions/2.2.4/lib/ruby/gems/2.2.0/gems/middleman-google-analytics-2.0.0/lib/middleman-google-analytics/helpers.rb:14:in `google_analytics_universal_tag'
```

It seems to be this method:

```
# Support for Middleman >= 3.4
def legacy_development?
  try(:development?) || try(:app).development?
end
```

**Comment by danielbayerlein** [comment]
@davidmles Ooops sorry! Which version (Middleman) do you use?

**Comment by davidmles** [comment]
The latest stable version, 3.4.1

```
    middleman (3.4.1)
      coffee-script (~> 2.2)
      compass (>= 1.0.0, < 2.0.0)
      compass-import-once (= 1.0.5)
      execjs (~> 2.0)
      haml (>= 4.0.5)
      kramdown (~> 1.2)
      middleman-core (= 3.4.1)
      middleman-sprockets (>= 3.1.2)
      sass (>= 3.4.0, < 4.0)
      uglifier (~> 2.5)
    middleman-core (3.4.1)
      activesupport (~> 4.1)
      bundler (~> 1.1)
      capybara (~> 2.4.4)
      erubis
      hooks (~> 0.3)
      i18n (~> 0.7.0)
      listen (~> 3.0.3)
      padrino-helpers (~> 0.12.3)
      rack (>= 1.4.5, < 2.0)
      thor (>= 0.15.2, < 2.0)
      tilt (~> 1.4.1, < 2.0)
```

**Comment by danielbayerlein** [comment]
@davidmles middleman-google-analytics v2.0.0 works for me with a clean Middleman 3.4.1 project. :confused: Is your project on :octocat:?

**Comment by davidmles** [comment]
No it's not. How could I debug it?

**Comment by danielbayerlein** [comment]
@davidmles I found the mistake! Sorry, i will fix the bug asap.

**Comment by danielbayerlein** [comment]
@davidmles I've published v2.0.1

**Comment by davidmles** [comment]
Thank you!
